gatorgrouper/tests/test_num_group.py

```diff
@@ -0,0 +1,53 @@
+"""Testing number of groups"""
```

Comment:
This file needs another test, where you actually check a real group number. Additionally, `test_check_number_groups_size_zero` likely should get a false result from `check_valid_num_group`, since zero groups is not possible to handle. These tests should be reviewed.

Reply:
Hi @Michionlion, `test_check_number_groups_size_zero` has the correct assertion due to that `--num-group 0` is able to be handled, which it should not. I will let @reibelj to take care of this in another PR since this is the part of Issue https://github.com/GatorEducator/gatorgrouper/issues/184 and he is assigned to `parse_arguments`.
Additionally, what do you mean by "check a real group number"?

Reply:
You should check a number like `2`, since all of your tests test exceptional conditions in some form. You should also ensure a normal condition works, and a group size of `2` or `3` would make sense for that.

Reply:
Thanks, will do.

Reply:
Solved by https://github.com/GatorEducator/gatorgrouper/pull/185/commits/72ea4950e5dcae701459351ccfcdf1947beb2453